flutter-studio/src/io/flutter/profiler/GcDurationData.java

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright 2018 The Chromium Authors. All rights reserved.
+ * Use of this source code is governed by a BSD-style license that can be
+ * found in the LICENSE file.
+ */
+package io.flutter.profiler;
+
+import com.android.tools.adtui.model.DefaultDurationData;
+
+public final class GcDurationData extends DefaultDurationData {
+   public GcDurationData(long duration) {
+      super(duration);
+   }
+
+   @Override
+   public String toString() {
+     return "GC Event";
```

Comment:
This toString seems odd. Perhaps remove or include the duration in the toString.

Reply:
Gone.